packages/cli/lib/cliUtils.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
+const fs = require('fs');
+
+const { error } = require('./print');
+const webpackConfig = require('../webpack/config');
+
+const IMA_CONF_FILENAME = 'ima.conf.js';
+
+function loadImaConf(rootDir) {
```

Comment:
The method should be async for allowing dynamic import *.mjd files in the future.

Reply:
Fixed Fixed